### ISSUE TITLE
[dev]confignetwork does not support capital letter in nictypes

### ISSUE THREAD

**Issue by bybai** [xcat2/xcat-core#3042]
confignetwork does not support capital letter in nictypes;
For example, customer always use nictypes.eth1=Ethernet, confignics support it , but confignetwork does not support it. In nics table, we support capital letter in nictypes,for example, Ethernet ,ethernet, BOND,bond etc. So it is must to fix it.
```
[root@byrh13 postscripts]# lsdef bytest15 |sed 's/    //g'|grep "^nic"
nicips.eth1=10.4.41.15
nicnetworks.eth1=10_0_0_0-255_0_0_0
nictypes.eth1=Ethernet

[root@byrh13 postscripts]# updatenode bytest15 confignetwork
bytest15: xcatdsklspost: downloaded postscripts successfully
bytest15: Fri May 12 07:19:49 UTC 2017 Running postscript: confignetwork
bytest15: [I]: NetworkManager is inactive.
bytest15: [I]: All valid nics and device list:
bytest15: [I]: eth1
bytest15: [I]: [E]: Error:   pair is invalid nic and nicdevice pair.
bytest15: ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
bytest15: configure nic and its device : eth1
bytest15: [E]: Error : please check nictypes for eth1.
bytest15: ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
bytest15: configure nic and its device : [E]: Error:   pair is invalid nic and nicdevice pair.
bytest15: [E]: Error : please check nictypes for [E]: Error:   pair is invalid nic and nicdevice pair..
bytest15: postscript: confignetwork exited with code 0
bytest15: Running of postscripts has completed.
```

**Comment by bybai** [comment]
fixed in #3043

**Comment by neo954** [comment]
Verified with the latest xCAT daily build version `2.13.4-snap201705120615`.
